.github/workflows/docker.yaml

```diff
@@ -0,0 +1,42 @@
+name: docker
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches:
+      - 'main'
+    paths:
+      - "**.go"
+      - "**.yaml"
+      - "!capm.yaml"
+      - "!osc-secret.yaml"
+      - "!example/**.yaml"
+      - "!helm/**"
+      - "Makefile"
+      - "!docs/src/**"
```

Comment:
this should also be in `pull_request` section

Reply:
Ok done (thanks)